src/shared/scrapers/AUS/index.js

```diff
@@ -28,8 +39,14 @@ const scraper = {
       const $tr = $(tr);
       const state = parse.string($tr.find('td:first-child').text());
       const cases = parse.number($tr.find('td:last-child').text());
+
+      const stateMapped = countryLevelMap[state];
+      if (!stateMapped) {
```

Comment:
NaBD but can do assert(stateMapped, message here)

Reply:
Thanks, fixed it.